upload/admin/language/en-gb/extension/payment/paypro_sofort.php

```diff
@@ -2,4 +2,4 @@
 
 require(DIR_SYSTEM . 'library/paypro/language/english.php');
 
-$_['heading_title'] = $_['heading_title_sofort_digital'];
+$_['heading_title'] = $_['heading_title_sofort'];
```

Comment:
We should update the translation keys in translation file

Reply:
done :)